app/minutes/test_minutes.py

```diff
@@ -222,15 +222,15 @@ def test_get_minute_doesnt_exist(self):
         received = self.socketio.get_received()
         assert received[0]["args"][0] == Response.MinuteDoesntExist
 
-    def test_get_minute_no_user(self):
+    def test_get_private_minute_no_user(self):
```

Comment:
I update the name of the test because the minute object that was initialized at the top of this file is private. Since the minute is private and the user is not logged in, they should get a permission error.